climbStairs.py

```diff
@@ -0,0 +1,36 @@
+class Solution(object):
```

Comment:
Can  you add comment to the link of the problem and description? (for all your questions)